### PR TITLE
Update repositories.txt

### PR DESCRIPTION
**Pull request by marcinsaj**

### CODE REVIEW

**Comment by per1234** [comment]
Hello @marcinsaj. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

**Review by umbynos — APPROVED** [review]

**Comment by umbynos** [comment]
Done:
```
2022/11/09 09:44:07 Loaded 5296 libraries from DB
Changing URL of library RTC_RX8025T from https://github.com/marcinsaj/RTC-RX8025T.git to https://github.com/marcinsaj/RTC_RX8025T.git
Success!
```

**Pull request merged**
